setup.py

```diff
@@ -18,22 +18,36 @@
 with open("requirements.txt") as requirements_file:
     requirements = requirements_file.read()
 
+
+def package_files(directory):
+    # source: https://stackoverflow.com/a/36693250
+    paths = []
+    for (path, directories, filenames) in os.walk(directory):
+        for filename in filenames:
+            paths.append(os.path.join(path, filename))
+    return paths
+
+package_file_list1 = package_files("examples")
+package_file_list2 = package_files("templates")
```

Comment:
Here we are in the directory in which setup.py resides. Hence, nothing will be found. (Did you forget to delete the old dirs locally?)